backend.py

```diff
@@ -15,6 +16,7 @@ def print_date(request):
 def main():
     settings = {}
     settings['service_name'] = 'backend'
+    settings['zipkin.encoding'] = Encoding.V2_JSON
```

Comment:
wonder if transport.py could become a function of settings..

ex configure_transport(settings) which then sets the encoding and the handler?